<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>728</width>
    <height>853</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Open-BLDC qSim</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="title">
       <string>Current</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="SignalPlot" name="signalPlot" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="simSettingsGroupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>200</width>
        <height>200</height>
       </size>
      </property>
      <property name="title">
       <string>Sim Settings</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QGroupBox" name="PWMSettingsGroupBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>PWM</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Frequency</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDoubleSpinBox" name="PWMFreqSpinBox">
            <property name="suffix">
             <string>kHz</string>
            </property>
            <property name="minimum">
             <double>1.000000000000000</double>
            </property>
            <property name="maximum">
             <double>9.990000000000000</double>
            </property>
           </widget>
          </item>
          <item row="2" column="0" rowspan="2">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Duty</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QDoubleSpinBox" name="PWMDutySpinBox">
            <property name="suffix">
             <string>%</string>
            </property>
            <property name="maximum">
             <double>99.989999999999995</double>
            </property>
            <property name="singleStep">
             <double>1.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="3" column="1" colspan="2">
           <widget class="QSlider" name="PWMDutySlider">
            <property name="maximum">
             <number>999</number>
            </property>
            <property name="value">
             <number>750</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QToolButton" name="PWMDutySendButton">
            <property name="toolTip">
             <string>Send Value</string>
            </property>
            <property name="text">
             <string>S</string>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QToolButton" name="PWMDutySendLockButton">
            <property name="toolTip">
             <string>Autosend Value Lock</string>
            </property>
            <property name="text">
             <string>L</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QToolButton" name="toolButton_3">
            <property name="toolTip">
             <string>Send Value</string>
            </property>
            <property name="text">
             <string>S</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QToolButton" name="toolButton_4">
            <property name="toolTip">
             <string>Autosend Value Lock</string>
            </property>
            <property name="text">
             <string>L</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>728</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
   </widget>
   <widget class="QMenu" name="menuSim">
    <property name="title">
     <string>Sim</string>
    </property>
    <addaction name="actionRunSim"/>
    <addaction name="actionPauseSim"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSim"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionRunSim"/>
   <addaction name="actionPauseSim"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionRunSim">
   <property name="icon">
    <iconset resource="main.qrc">
     <normaloff>:/icons/run_sim_active.png</normaloff>
     <disabledoff>:/icons/run_sim_inactive.png</disabledoff>:/icons/run_sim_active.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Run</string>
   </property>
   <property name="toolTip">
    <string>Run Simulator</string>
   </property>
  </action>
  <action name="actionPauseSim">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="main.qrc">
     <normaloff>:/icons/pause_sim_active.png</normaloff>
     <disabledoff>:/icons/pause_sim_inactive.png</disabledoff>:/icons/pause_sim_active.png</iconset>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
   <property name="toolTip">
    <string>Pause Simulator</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>SignalPlot</class>
   <extends>QWidget</extends>
   <header>signalplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="main.qrc"/>
 </resources>
 <connections/>
</ui>
